CHANGELOG.md

```diff
@@ -28,6 +28,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (`JxlDecoderDCOutBufferSize` and `JxlDecoderSetDCOutBuffer`) and will be
   removed in the next release.
 
+## [0.5] - 2021-07-??
```

Comment:
This goes in the Unreleased section above

Reply:
Done (so, removed mention of 0.5, let me know if the intention is to change the word "Unreleased" into "0.5")

Reply:
We haven't release 0.5 yet, only branched v0.5.x. This is a retroactive documentation change only.
Once we release 0.5 (like, right before we do so) we will know the release date and update the header.